Lib/test/test_enum.py

```diff
@@ -2321,7 +2321,7 @@ class OkayEnum(CustomStrEnum):
         self.assertEqual(str(OkayEnum.one), 'one')
         with self.assertWarns(DeprecationWarning):
             self.assertEqual('{}'.format(OkayEnum.one), '1')
-        self.assertEqual(OkayEnum.one, '{}'.format(OkayEnum.one))
+            self.assertEqual(OkayEnum.one, '{}'.format(OkayEnum.one))
```

Comment:
This indeed resolves the warning, but I wonder if this assertion is useful at all since we already check that `'{}'.format(OkayEnum.one) == '1'` (the line above) and that `OkayEnum.one == '1'` (3 lines about). Perhaps it can just be removed?